### PR TITLE
Fix the tests

### PR DESCRIPTION
**Pull request by adamduren**

### CODE REVIEW

File: src/@ionic-native/plugins/google-maps/index.ts [hunk 1]
```diff
@@ -2694,7 +2694,7 @@ export class GoogleMap extends BaseClass {
             }
             clearInterval(timer);
             this._objectInstance.remove();
-            if (targets.length > 0 && targets[0] && targets[0].offsetWidth < 100 || targets[0].offsetHeight < 100) {
+            if (targets.length > 0 && targets[0] && (targets[0].offsetWidth < 100 || targets[0].offsetHeight < 100)) {
```
Comment (adamduren) [inline src/@ionic-native/plugins/google-maps/index.ts]: It looks like this fix was somehow reverted. Restoring the tests caught the bug.

File: package.json [hunk 2]
```diff
@@ -63,19 +66,6 @@
     "winston": "^3.2.1",
     "zone.js": "^0.8.29"
   },
-  "jest": {
```
Comment (adamduren) [inline package.json]: Because we have `jest.config.js` this is redundant and not used.

**Comment by wf9a5m75** [comment]
It works perfectly! Thank you for your work @adamduren !
```
masashis-mini:ionic-native-google-maps-master masashi$ npm run test

> @ionic-native/google-maps@5.5.0 pretest /Users/masashi/Documents/workspace2/ionic-native-google-maps-master
> npm run prebuild; npm run build:esm


> @ionic-native/google-maps@5.5.0 prebuild /Users/masashi/Documents/workspace2/ionic-native-google-maps-master
> rimraf -rf dist


> @ionic-native/google-maps@5.5.0 build:esm /Users/masashi/Documents/workspace2/ionic-native-google-maps-master
> ts-node -P scripts/tsconfig.json scripts/tasks/build-esm

info: transformClass: LatLng {"durationMs":1}
info: transformClass: LatLngBounds {"durationMs":1}
info: transformClass: VisibleRegion {"durationMs":1}
info: transformClass: GoogleMaps {"durationMs":0}
info: transformClass: BaseClass {"durationMs":1}
info: transformClass: BaseArrayClass {"durationMs":1}
info: transformClass: Circle {"durationMs":0}
info: transformClass: Environment {"durationMs":1}
info: transformClass: Geocoder {"durationMs":0}
info: transformClass: LocationService {"durationMs":0}
info: transformClass: Encoding {"durationMs":0}
info: transformClass: Poly {"durationMs":0}
info: transformClass: Spherical {"durationMs":0}
info: transformClass: StreetViewPanorama {"durationMs":0}
info: transformClass: GoogleMap {"durationMs":1}
info: transformClass: GroundOverlay {"durationMs":0}
info: transformClass: HtmlInfoWindow {"durationMs":0}
info: transformClass: Marker {"durationMs":0}
info: transformClass: MarkerCluster {"durationMs":1}
info: transformClass: Polygon {"durationMs":0}
info: transformClass: Polyline {"durationMs":0}
info: transformClass: TileOverlay {"durationMs":0}
info: transformClass: KmlOverlay {"durationMs":1}

> @ionic-native/google-maps@5.5.0 test /Users/masashi/Documents/workspace2/ionic-native-google-maps-master
> jest && npm run lint

 PASS  src/@ionic-native/_core/decorators/common.spec.ts
 PASS  src/@ionic-native/plugins/google-maps/google-map.spec.ts
  ● Console

    console.error dist/@ionic-native/plugins/google-maps/index.js:1170
      [GoogleMaps]You need to execute "$> ionic cordova run browser"."$> ionic serve" is not supported.
    console.warn node_modules/@ionic-native/core/decorators/common.js:276
      Native: tried accessing the GoogleMaps plugin but it's not installed.
    console.error dist/@ionic-native/plugins/google-maps/index.js:1170
      [GoogleMaps]cordova-plugin-googlemaps is not installed or not ready yet.


Test Suites: 2 passed, 2 total
Tests:       21 passed, 21 total
Snapshots:   8 passed, 8 total
Time:        2.358s, estimated 7s
Ran all test suites.

> @ionic-native/google-maps@5.5.0 lint /Users/masashi/Documents/workspace2/ionic-native-google-maps-master
> gulp lint

[12:53:57] Using gulpfile ~/Documents/workspace2/ionic-native-google-maps-master/gulpfile.js
[12:53:57] Starting 'lint'...
Warning: The 'await-promise' rule requires type information.
Warning: The 'deprecation' rule requires type information.
[12:53:57] Finished 'lint' after 776 ms
masashis-mini:ionic-native-google-maps-master masashi$ 
```

**Comment by wf9a5m75** [comment]
@adamduren I sent you a message from your website. Please check it out.

**Pull request merged**